assets/css/tailwind.css

```diff
@@ -1,3 +1,10 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
```

Comment:
had two choices, either but base styles in here or in the json config. Figured its better to use this file for base styles.

Reply:
Nit: Might want to consider differentiating h3 and h4.

Reply:
also these header styles are just temporary until we have some designs

Reply:
I just noticed my comment was added after yours when I refreshed 😅  
Let's update these later when Kei has css rules for our headers

Reply:
updated